index.html

```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
                     <img src="./images/angelist.png" alt="angelist"/>
                 </a>
             </li>
-
-        </ul>
+                  </ul>
+        </div>
+    </section>
+    <section class="work">
+<div class="image">
+    <img src="./images/snap1.png" alt="snap1"/>
+</div>
+<div class="content">
+    <h2>Tonic</h2>
+    <div class="event2-tonic">
+        <div class="word">
+          <h3>CANOPY</h3>
+        </div>
+        <img src="./images/Counter.png" alt="counter image" />
+        <div class="role">
+          <h3>Back End Dev</h3>
         </div>
+        <img src="./images/Counter.png" alt="counter image" class="dot" />
+        <div class="year">
+          <h3>2015</h3>
+        </div>
+      </div>
+    </div>
+    <p class="disc">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="tonicl">
+        <ul class="bottom-list">
+            <li><button class="language-buttons" type="button">html</button></li>
+            <li><button class="language-buttons" type="button">css</button></li>
+            <li><button class="language-buttons" type="button">javaScript</button></li>
+        </ul>
+    
+	          <div class="btn">
+	            <button type="submit" class="see">See projects</button>
+	          </div>
+            </ul>
+</div>
     </section>
```

Comment:
- [ ] Kindly edit the alignment of the list of technologies and the button of each project card in the work section. You are expected to stick to the design template as much as possible.
 ### Your design
![1](https://user-images.githubusercontent.com/94551484/192068746-a3b9b5f3-e137-4958-aa84-c50c4c37bdac.png)
### Figma template
![f1](https://user-images.githubusercontent.com/94551484/192068751-3ae25852-d70d-4559-9d93-0527462da13e.png)


- [ ] Kindly reduce the spacing between project cards to match the Figma template.
### YOur design
![2](https://user-images.githubusercontent.com/94551484/192068819-268c1733-099a-41b0-b620-4f0bf6922423.png)
### Figma template
![f2](https://user-images.githubusercontent.com/94551484/192068830-d4f4dcb0-1215-4a53-8078-95cf81118d6a.png)